CC midplane refinement (#801)
Deep-MI/FastSurfer: doc/scripts/fastsurfer_cc.rst

   :func: make_parser
   :prog: fastsurfer_cc.py

Midplane extraction
-------------------

> The ``--midplane_method`` flag controls how the corpus callosum pipeline refines the midsagittal plane before segmentation. The current implementation lives in ``CorpusCallosum/registration/midsagittal_plane_alignment.py``.

Reviewer: ```suggestion
The ``--midplane_method`` flag controls how the corpus callosum pipeline refines the midsagittal plane before segmentation. This is implemented in ``CorpusCallosum/registration/midsagittal_plane_alignment.py``.
```

Author: applied, thanks!

> When the corpus callosum segmentation does not align well with the mid-sagittal plane, can be changed for better results.

Reviewer: ```suggestion
When the corpus callosum segmentation does not align well with the mid-sagittal plane, this option can be changed for better results.
```

Author: applied, thanks!

Available modes are:

> - ``fsaverage_symmetry``: (default) Align to fsaverage, then search for a small left-right shift that minimizes mirrored aseg-label mismatch near the midline

Reviewer: Maybe indicate what was used in the FastSurfer-CC paper?

Author: I did that in the description
